index.html

```diff
@@ -106,6 +106,7 @@
 				<li><a href="http://samtools.sourceforge.net/SAM1.pdf">SAM Format Specification</a></li>
 				<li><a href="http://sourceforge.net/mail/?group_id=246254">SAMTools mailing Lists</a></li>
 				<li><a href="http://samtools.sourceforge.net/">SAMTools Home Page</a></li>
+				<li><a href="tools-that-use-picard.html">Tools that use Picard</a></li>
```

Comment:
No offense to the author, but I agree that we should not be in the business of compiling lists of external tools that use or wrap Picard -- that's a user support minefield.